FindChannelFunction/src/main/java/no/unit/nva/channel/model/outgoing/Channel.java

```diff
@@ -0,0 +1,62 @@
+package no.unit.nva.channel.model.outgoing;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Channel {
+
+    @JsonProperty("title")
+    private String originalTitle;
+
+    @JsonProperty("onlineIssn")
+    private String onlineIssn;
```

Comment:
e-issn is a relatively standardised concept, and the other is issn. I suspect that spelling this out in this way is fair enough, there is no ambiguity

Reply:
Speiler bare hva kanalregisteret kaller det. Vi får eventuelt ta det opp med produkteierne om vi ønsker andre navn her.